src/Generator.php

```diff
@@ -291,4 +296,55 @@ protected function error($string)
             echo $string . "\r\n";
         }
     }
+
+    /**
+     * Add all macroable classes which are not already loaded as an alias and have defined macros.
+     *
+     * @param Collection $aliases
+     */
+    protected function addMacroableClasses(Collection $aliases)
+    {
+        $macroable = $this->getMacroableClasses($aliases);
+
+        foreach ($macroable as $class) {
+            $reflection = new ReflectionClass($class);
+
+            if (!$reflection->getStaticProperties()['macros']) {
+                continue;
+            }
+
+            $aliases[] = new Alias($this->config, $class, $class, [], $this->interfaces);
+        }
+    }
+
+    /**
+     * Get all loaded macroable classes which are not loaded as an alias.
+     *
+     * @param Collection $aliases
+     * @return Collection
+     */
+    protected function getMacroableClasses(Collection $aliases)
+    {
+        return collect(get_declared_classes())
+            ->filter(function ($class) {
+                $reflection = new ReflectionClass($class);
+
+                // Filter out internal classes and class aliases
+                return !$reflection->isInternal() && $reflection->getName() == $class;
+            })
+            ->filter(function ($class) {
+                $traits = class_uses($class);
+
+                // Filter only classes with the macroable trait
+                return isset($traits['Illuminate\Support\Traits\Macroable']);
```

Comment:
Please use `Illuminate\Support\Traits\Macroable::class` and also import it, thanks!